site/content/docs/main/contributions/tencent-config.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "Use Tencent Cloud Object Storage as Velero's storage destination."
+layout: docs
+---
+
+
+You can deploy Velero on Tencent [TKE](https://console.cloud.tencent.com/tke2/) , or even on any other Kubernetes cluster, but anyway you can use Tencent Cloud Object Store as a destination for Velero’s backups.
+
+
+## Prerequisites
+
+- Registered [Tencent Cloud Account](https://cloud.tencent.com/register).
+- Tencent Cloud COS service, referred to as COS, has been launched.
+- A Kubernetes cluster has been created, cluster version v1.10 or later, and the cluster can use DNS and Internet services normally. If you need to create a TKE cluster, Please refer to [Create a cluster](https://cloud.tencent.com/document/product/457/32189).
```

Comment:
suggestion: "If you need to create a TKE cluster, refer to the Tencent [create a cluster](https://cloud.tencent.com/document/product/457/32189) documentation."

Reply:
thank u, I will fixed it.